Specs/DateTime/English/DateTimeModelComplexCalendar.json

```diff
@@ -2300,5 +2300,237 @@
         }
       }
     ]
+  },
+  {
+    "Input":
+      "Cortana, can you block time for us to meet tomorrow 12PM Singapore time for a quick 30min conference call.",
+    "Context": {
+      "ReferenceDateTime": "2018-06-12T00:00:00"
+    },
+    "NotSupported": "javascript,python",
+    "Results": [
+      {
+        "Text": "tomorrow 12pm singapore time",
+        "Start": 43,
+        "End": 70,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2018-06-13T12",
+              "type": "datetime",
+              "timezone": "UTC+08:00",
+              "timezoneText": "singapore time",
+              "utcOffsetMins": "480",
+              "value": "2018-06-13 12:00:00"
+            }
+          ]
+        }
+      },
+      {
+        "Text": "30min",
+        "Start": 84,
+        "End": 88,
+        "TypeName": "datetimeV2.duration",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "PT30M",
+              "type": "duration",
+              "value": "1800"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input":
+      "Cortana, schedule a skype meeting with Prachi for Tuesday May 22 between 9:30 and 3:00 pm PST.",
+    "Context": {
+      "ReferenceDateTime": "2018-06-12T00:00:00"
+    },
+    "NotSupported": "javascript,python",
+    "Results": [
+      {
+        "Text": "tuesday may 22 between 9:30 and 3:00 pm pst",
+        "Start": 50,
+        "End": 92,
+        "TypeName": "datetimeV2.datetimerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(XXXX-05-22T09:30,XXXX-05-22T15:00,PT5H30M)",
+              "type": "datetimerange",
+              "timezone": "UTC-08:00",
+              "timezoneText": "pst",
+              "utcOffsetMins": "-480",
+              "start": "2018-05-22 09:30:00",
+              "end": "2018-05-22 15:00:00"
+            },
+            {
+              "timex": "(XXXX-05-22T09:30,XXXX-05-22T15:00,PT5H30M)",
+              "type": "datetimerange",
+              "timezone": "UTC-08:00",
+              "timezoneText": "pst",
+              "utcOffsetMins": "-480",
+              "start": "2019-05-22 09:30:00",
+              "end": "2019-05-22 15:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input":
+      "Cortana, can you please schedule a meeting with Maria either Monday or Tuesday next week between 1pm and 2pm CST",
```

Comment:
Ideally entities here should be extracted as datetimealt. Will this come in a next PR?